src/init/mod.rs

```diff
@@ -193,9 +193,17 @@ fi"#,
         Some(shell_basename) => {
```

Comment:
Since you're touching this it might make sense to also combine the `None` and `Some(shell_basename)` match branches.

Reply:
> Is there a way to put a constraint on the shell arg using clap?

Seems there is https://docs.rs/clap/2.33.3/clap/struct.Arg.html#examples-38